### PR TITLE
Settings Button + Window + Icon

### PR DESCRIPTION
**Pull request by yageljaoui**
need to fix formatting from " to '

### CODE REVIEW

File: src/Components/Settings/SettingsWindow.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import GeneralWindow from "Components/GeneralComponents/GeneralWindow";
+
+export default function SettingsWindow({ unmountSettingsWindow }) {
+	return (
+		<GeneralWindow unmountGeneralWindow={unmountSettingsWindow}>
+			<form>
+				<br />
+				<label htmlFor='masterVolume'>Master</label>
+				<br />
+				<input type='range' min='0' max='100' value='73' step='1' name='masterVolume' id='masterVolume' />
```
Comment (elShailee) [inline src/Components/Settings/SettingsWindow.jsx:11]: all "value" field should be replaced to "defaultValue"

File: src/Components/Settings/SettingsButton.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#settingsButton {
+	width: 3.125vw;
+	height: 3.125vw;
+	background-color: #778ca1;
+	border: 1px solid #101010;
+}
```
Comment (elShailee) [inline src/Components/Settings/SettingsButton.css:6]: - please change 3.125vw to `calc(50vh/9)`
 - please add actionsbutton to the upperrightcontainer, and define the margin as 0.5 of the height unit vlaue.
 - recommended to use a css variable of height-unit for this.

File: src/Components/AllTheFloatingWindows/AllTheFloatingWindows.js [hunk 3]
```diff
@@ -1,28 +1,35 @@
...
+		<SettingsWindow unmountSettingsWindow={() => dispatch(unmountComponents(["SettingsWindow"]))} />
+	);
+
+	const ActionsWindowAsVar = <ActionsWindow unmountActionsWindow={() => dispatch(unmountComponents(["ActionsWindow"]))} />;
 	return (
 		<div>
 			{isCoinsPurchaseWindowMounted && CoinsPurchaseWindowAsVar}
 			{isEnergyPurchaseWindowMounted && EnergyPurchaseWindowAsVar}
 			{isActionsWindowMounted && ActionsWindowAsVar}
+			{isSettingsWindowMounted && SettingsWindowAsVar}
```
Comment (elShailee) [inline src/Components/AllTheFloatingWindows/AllTheFloatingWindows.js:32]: please maintain an order between to components

File: src/Components/Settings/SettingsButton.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	};
+
+	const settingsIcon = <img src={SettingsIcon} alt='' className='bigIcons' />;
+
+	return (
+		<div id='settingsButtonContainer'>
+			<GeneralButton id='settingsButton' outlined onButtonClick={onSettingsButtonClick}>
+				{settingsIcon}
+			</GeneralButton>
+			{isSettingsWindowOpen && <SettingsWindow />}
```
Comment (elShailee) [inline src/Components/Settings/SettingsButton.jsx:28]: please remove, not-needed

File: src/Components/Settings/SettingsWindow.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+				<hr />
+				<br />
+				<label htmlFor='masterVolume'>Music</label>
+				<br />
+				<input type='range' min='0' max='100' value='32' step='1' name='masterVolume' id='masterVolume' />
+				<hr />
+				<br />
+				<label htmlFor='masterVolume'>SFX</label>
+				<br />
+				<input type='range' min='0' max='100' value='55' step='1' name='masterVolume' id='masterVolume' />
```
Comment (elShailee) [inline src/Components/Settings/SettingsWindow.jsx:21]: please give each a unique name and id

File: src/GeneralStyles/iconsStyles.css [hunk 6]
```diff
@@ -1,9 +1,14 @@
...
-  width: 19px;
-  height: 19px;
+	width: 19px;
+	height: 19px;
+}
+
+.bigIcons {
+	width: 45px;
+	height: 45px;
 }
```
Comment (elShailee) [inline src/GeneralStyles/iconsStyles.css:14]: please remove

File: src/Redux/Slices/MountedComponents.js [hunk 7]
```diff
@@ -1,17 +1,19 @@
...
+import { errorMessagesTexts } from "Texts/gameplayTexts";
+import { isAnObject } from "Utils/utilFuncs";
 
 const initialState = {
 	ActionsTab: false,
 	ActionsWindow: false,
+	SettingsWindow: false,
 	EnergyPurchaseWindow: false,
 	CoinsPurchaseWindow: false,
+	SettingsTab: false,
```
Comment (elShailee) [inline src/Redux/Slices/MountedComponents.js:11]: please remove

File: src/Screens/GameplayScreen/GameplayScreen.css [hunk 8]
```diff
@@ -1,29 +1,36 @@
...
+	font-family: Helvetica;
 }
 
 /*-------------------
   Button Containers
 -------------------*/
 
+#upperRightButtonsContainer {
+	position: absolute;
+	left: 78vw;
```
Comment (elShailee) [inline src/Screens/GameplayScreen/GameplayScreen.css:11]: please align to the right, symetric to `upperLeftButtonsContainer`
(right:20px)

File: src/Screens/GameplayScreen/GameplayScreen.js [hunk 9]
```diff
@@ -22,6 +23,10 @@ export default function GameplayScreen() {
 				<CoinsButton />
 			</div>
 
+			<div id='upperRightButtonsContainer'>
+				<SettingsButton />
+			</div>
+
 			<ActionsButton />
```
Comment (elShailee) [inline src/Screens/GameplayScreen/GameplayScreen.js:31]: please insert the actionsbutton to the container

File: src/Components/Settings/SettingsButton.css (hunk 2 above)
Comment (yageljaoui) [reply src/Components/Settings/SettingsButton.css]: I'm having issues converting the sizing and padding of the Settings Button to %s.

**Review by elShailee — APPROVED** [review]

**Pull request merged**